src/app/tl-mega-menu/tl-mega-menu.component.scss

```diff
@@ -2,6 +2,23 @@
 @import "../shared/styles/tl-variables.scss";
 
 :host {
+  .tl-mega-menu-custom-top {
+    width: calc(var(--wvr-dropdown-menu-width) + 6px);
+    margin-left: -19px;
```

Comment:
Is margin the right choice here? Should we be using padding?

Reply:
margin rule enabled the gap on the left to be occupied by the top-content